server/routes/cartRoutes.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import {
+  createCart,
+  deleteCart,
+  getMyCart,
+  removeFromCart,
+  updateCart,
+} from "../controllers/cartControllers.js";
+import { verifyJWT } from "../middleware/verifyJWT.js";
+import { validateCart } from "../middleware/validate.js";
+const router = express();
+
+router.post("/", verifyJWT, validateCart, createCart);
+router.delete("/items", verifyJWT, removeFromCart);
```

Comment:
instead of adding the productId to the payload, add it to the params. like this `/items/:productId`

Reply:
done